compatibility/README.md

```diff
@@ -31,11 +32,12 @@ Version: [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.
 implementation group: 'com.foursoft.harness.compatibility', name: 'compatibility-core', version: 'VERSION'
 ```
 
-### Compatibility VEC 1.1.X to VEC 1.2.X
+### Compatibility VEC 1.1.3 to VEC 1.2.X
```

Comment:
Technically, there is just the 1.1.3 but I think we should still not specify a patch version here, I believe it's also not specified in the code.

Reply:
well, sure. All wrappers work directly on "com.foursoft.harness.vec.v113" package. That should be clear in the README, too.

Reply:
I believe we later defined that as a flaw, hence why we used "v12x" and "v2x" for further package names. We kind of added that already to the ReadMe of the VEC module, that VEC 1.1.3 is the only version within 1.1.X. So I think that's enough in making it clear. However in the compatibility module, we never specified 1.1.3 anywhere and I think that's fine.